src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImpl.java

```diff
@@ -219,20 +219,24 @@ private void rejectHouseholdIndividual(CaseContainerDTO caseDetails) {
     }
   }
 
+  private void rejectHouseholdIndividual(CaseDTO caseDetails) {
```

Comment:
Yes, that did occur to me but it meant changing other methods that call it using CaseContainerDTO. However, since you mentioned it, I've taken another look at those methods and noticed that they all mat the returned CaseContainerDTO to a CaseDTO anyhow. So I've changed it and them now.